Nautilus/Utility/AudioUtils.cs

```diff
@@ -14,6 +14,19 @@ namespace Nautilus.Utility;
 /// </summary>
 public static partial class AudioUtils
 {
+    /// <summary>
+    /// 3D sounds
+    /// </summary>
+    public const MODE k3DOptiSoundModes = MODE.DEFAULT | MODE._3D | MODE.ACCURATETIME | MODE._3D_LINEARSQUAREROLLOFF;
```

Comment:
All of these consts have somewhat odd names. They're not terrible but it does cause compilation warnings for me. Could be changed.